File: 666-service/src/main/java/com/service/PingService.java
package com.service;
import java.util.List;

import pojo.TbPing;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface PingService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbPing> findAll();
	
	//返回今日
	public List<TbPing>findToday();

	
	/**
	 * 增加
	*/
	public void add(TbPing ping);
	
	
	/**
	 * 修改
	 */
	public void update(TbPing ping);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbPing findOne(Integer id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Integer[] ids);


}

File: 666-web/src/main/webapp/jsp/AnglaJS/js/service/pingService.js
//服务层
app.service('pingService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../ping/findAll');
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../ping/findPage?page='+page+'&rows='+rows);
	}

	//今日
	this.findToday=function () {
		return $http.get('../ping/today');
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../ping/findOne?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../ping/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../ping/update',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../ping/delete?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../ping/search?page='+page+"&rows="+rows, searchEntity);
	}    	
});
